src/App.vue

```diff
@@ -109,6 +112,15 @@ export default {
       // Then check if btc is operational
       if (this.loadingProgress <= 60) {
         this.loadingText = "Loading Bitcoin Core...";
+
+        // Warn users against pulling power if Core is taking a while
+        const bitcoinSlowDelay = 10 * SECONDS_IN_MS;
+        if (!this.bitcoinStarted) {
+          this.bitcoinStarted = Date.now();
+        } else if (Date.now() - this.bitcoinStarted > bitcoinSlowDelay) {
+           this.loadingText += " This can take a while, don't pull the power!";
```

Comment:
There's a small chance that "don't pull the power" might not be obvious to non-native English speakers. Shall we change it to "please don't turn off your Umbrel" to remove any kind of ambiguity?

Reply:
Good idea.